src/main/java/com/exadel/discountwebapp/vendor/mapper/VendorMapper.java

```diff
@@ -25,7 +26,7 @@ public VendorMapper(LocationService locationService) {
 
     public VendorResponseVO toVO(Vendor vendor) {
```

Comment:
please add tests to check that mapping is correct

Reply:
added